Usage in Angular 16 app results in: Cannot find module 'chord_sheet/item' or its corresponding type declarations.

Importing `ChordSheetJS` in my Angular 16 app:
```
import ChordSheetJS from "chordsheetjs";
```
 results in error: 
```

Error: node_modules/chordsheetjs/lib/main.d.ts:1419:34 - error TS2307: Cannot find module 'chord_sheet/item' or its corresponding type declarations.

1419     isEvaluatable: (item: import("chord_sheet/item").default) => boolean;
                                      ~~~~~~~~~~~~~~~~~~


Error: node_modules/chordsheetjs/lib/main.d.ts:1420:38 - error TS2307: Cannot find module 'chord_sheet/item' or its corresponding type declarations.

1420     isChordLyricsPair: (item: import("chord_sheet/item").default) => boolean;
                                          ~~~~~~~~~~~~~~~~~~


Error: node_modules/chordsheetjs/lib/main.d.ts:1422:26 - error TS2307: Cannot find module 'chord_sheet/item' or its corresponding type declarations.

1422     isTag: (item: import("chord_sheet/item").default) => boolean;
                              ~~~~~~~~~~~~~~~~~~


Error: node_modules/chordsheetjs/lib/main.d.ts:1430:29 - error TS2307: Cannot find module 'chord_sheet/chord_pro/evaluatable' or its corresponding type declarations.

1430     evaluate: (item: import("chord_sheet/chord_pro/evaluatable").default, metadata: Metadata, configuration: import("formatter/configuration/configuration").default) => string;
                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Error: node_modules/chordsheetjs/lib/main.d.ts:1430:117 - error TS2307: Cannot find module 'formatter/configuration/configuration' or its corresponding type declarations.

1430     evaluate: (item: import("chord_sheet/chord_pro/evaluatable").default, metadata: Metadata, configuration: import("formatter/configuration/configuration").default) => string;
                                                                                                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Error: node_modules/chordsheetjs/lib/main.d.ts:1459:38 - error TS2307: Cannot find module 'chord_sheet/item' or its corresponding type declarations.

1459         isEvaluatable: (item: import("chord_sheet/item").default) => boolean;
                                          ~~~~~~~~~~~~~~~~~~


Error: node_modules/chordsheetjs/lib/main.d.ts:1460:42 - error TS2307: Cannot find module 'chord_sheet/item' or its corresponding type declarations.

1460         isChordLyricsPair: (item: import("chord_sheet/item").default) => boolean;
                                              ~~~~~~~~~~~~~~~~~~


Error: node_modules/chordsheetjs/lib/main.d.ts:1462:30 - error TS2307: Cannot find module 'chord_sheet/item' or its corresponding type declarations.

1462         isTag: (item: import("chord_sheet/item").default) => boolean;
                                  ~~~~~~~~~~~~~~~~~~


Error: node_modules/chordsheetjs/lib/main.d.ts:1470:33 - error TS2307: Cannot find module 'chord_sheet/chord_pro/evaluatable' or its corresponding type declarations.

1470         evaluate: (item: import("chord_sheet/chord_pro/evaluatable").default, metadata: Metadata, configuration: import("formatter/configuration/configuration").default) => string;
                                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Error: node_modules/chordsheetjs/lib/main.d.ts:1470:121 - error TS2307: Cannot find module 'formatter/configuration/configuration' or its corresponding type declarations.

1470         evaluate: (item: import("chord_sheet/chord_pro/evaluatable").default, metadata: Metadata, configuration: import("formatter/configuration/configuration").default) => string;
                                                                                                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/package.json
+++ b/package.json
@@ -50,6 +50,7 @@
     "ts-jest": "^27.1.4",
     "ts-node": "^10.7.0",
     "ts-pegjs": "^3.0.0",
+    "tsc": "^2.0.4",
     "typescript": "^4.6.3"
   },
   "scripts": {
@@ -63,7 +64,8 @@
     "build:code-generate": "yarn build:suffix-normalize && yarn build:chord-suffix-grammar && yarn build:pegjs && yarn build:scales",
     "build:sources": "parcel build",
     "build:browserify": "browserify lib/index.js --outfile lib/bundle.js --standalone ChordSheetJS",
-    "build": "yarn build:code-generate && yarn build:sources && yarn build:browserify",
+    "build:check-types": "tsc lib/main.d.ts",
+    "build": "yarn build:code-generate && yarn build:sources && yarn build:browserify && yarn build:check-types",
     "dev": "parcel watch --no-cache",
     "jest": "jest",
     "test": "yarn lint && yarn jest",

--- a/src/index.ts
+++ b/src/index.ts
@@ -20,22 +20,6 @@ import TextFormatter from './formatter/text_formatter';
 import UltimateGuitarParser from './parser/ultimate_guitar_parser';
 
 import {
-  isEvaluatable,
-  isChordLyricsPair,
-  lineHasContents,
-  isTag,
-  isComment,
-  stripHTML,
-  each,
-  when,
-  hasTextContents,
-  lineClasses,
-  paragraphClasses,
-  evaluate,
-  fontStyleTag,
-} from './template_helpers';
-
-import {
   CHORUS,
   INDETERMINATE,
   NONE,
@@ -67,22 +51,7 @@ export { default as Tag } from './chord_sheet/tag';
 export { default as Ternary } from './chord_sheet/chord_pro/ternary';
 export { default as TextFormatter } from './formatter/text_formatter';
 export { default as UltimateGuitarParser } from './parser/ultimate_guitar_parser';
-
-export const templateHelpers = {
-  isEvaluatable,
-  isChordLyricsPair,
-  lineHasContents,
-  isTag,
-  isComment,
-  stripHTML,
-  each,
-  when,
-  hasTextContents,
-  lineClasses,
-  paragraphClasses,
-  evaluate,
-  fontStyleTag,
-};
+export { default as templateHelpers } from './template_helpers';
 
 export {
   CHORUS,
@@ -121,5 +90,4 @@ export default {
   TextFormatter,
   UltimateGuitarParser,
   VERSE,
-  templateHelpers,
 };

--- a/src/template_helpers.ts
+++ b/src/template_helpers.ts
@@ -81,3 +81,19 @@ export function fontStyleTag(font: Font) {
 
   return '';
 }
+
+export default {
+  isEvaluatable,
+  isChordLyricsPair,
+  lineHasContents,
+  isTag,
+  isComment,
+  stripHTML,
+  each,
+  when,
+  hasTextContents,
+  lineClasses,
+  paragraphClasses,
+  evaluate,
+  fontStyleTag,
+};

--- a/test/default_export.test.ts
+++ b/test/default_export.test.ts
@@ -24,7 +24,6 @@ describe('default export', () => {
     'INDETERMINATE',
     'VERSE',
     'NONE',
-    'templateHelpers',
   ].forEach((constantName) => {
     it(`contains ${constantName}`, () => {
       expect(typeof chordsheetjs[constantName]).not.toEqual('undefined');

